react-client/src/Components/App.js

```diff
@@ -111,63 +117,56 @@ function App() {
     injectDOMEventInBody();
   };
 
+ 
+
+  // const resetdom = () => setshowSwitch(true);
+
   return (
     <div>
       {/* website page renders here... */}
       {!PRODUCTION_MODE && <div id="samplePage"></div>}
 
-      <div>
-        {domInfo.map((domInfo) => {
-          const handleDelete = () => {
-            setDomInfo((prevNotes) =>
-              prevNotes.reduce(
-                (init, curr) =>
-                  curr.x === domInfo.x && curr.y === domInfo.y
-                    ? init
-                    : init.push(curr) && init,
-                []
-              )
+      <div onClick={switchdom}>{domSwitch ? <DomSwitch /> : null}</div>
```

Comment:
You can improve this code snippet by doing this: `{domSwitch && <DomSwitch />}`

Reply:
noted will edit this part

Reply:
Any updates

Reply:
Any updates

Reply:
tomorrow pa cgro ko ni ma update kay  gamiton haiza akoang laptop